packages/documentation-site/patternfly-docs/content/components/pagination/pagination.md

```diff
@@ -61,19 +61,25 @@ When it's difficult to calculate the total number of items or page count upfront
 
 ### Desktop
 
-For desktop views, place [full pagination](#full-pagination) above the content it controls, typically within a toolbar.
+For desktop views, place [full pagination](#full-pagination) above the content it controls ("top pagination"), typically within a toolbar. 
 
 <div class="ws-docs-content-img">
 ![an example of a desktop pagination component within a toolbar.](./img/toolbar-pagination.svg)
 </div>
 
 When used in a toolbar, pagination may leave limited room for other items like filters or bulk selectors. These additional items should automatically move into an [overflow menu](/components/toolbar/design-guidelines#overflow-menu) as needed.
 
-For components with long, scrolling content (like a table), you can add a second full pagination component below the content to ensure that users can navigate without scrolling back to the top.
+Also place full pagination below the component it controls ("bottom pagination").
+
+<div class="ws-docs-content-img">
+![an example of a bottom pagination component placed after a table.](./img/bottom-pagination.svg)
+</div>
+
+Bottom pagination lets users easily navigate lengthy datasets without scrolling back to the top, while ensuring that pagination adapts appropriately when switching from desktop to mobile.
 
 ### Mobile 
 
-Mobile pagination is formatted differently in order to conserve space. On mobile, full pagination will collapse to a pagination summary with an item count. [Learn how to customize overflow menu breakpoints in our toolbar design guidelines](/components/toolbar/design-guidelines).
+On mobile, pagination adapts in order to better conserve space. When switching from desktop to mobile, top pagination will collapse to a pagination summary with an item count and bottom pagination will adapt to a more compact design. [Learn how to customize overflow menu breakpoints in our toolbar design guidelines](/components/toolbar/design-guidelines).
```

Comment:
Just an opinion, but I would probably only link the "toolbar design guidelines" text? It's more concise (easier to read) and accurately describes the resulting page. I also wonder why it's here - there is no mention of doing anything with overflow menus? There is a line in the desktop section that may be a more appropriate spot for this link? https://github.com/patternfly/patternfly-org/blob/bfd0dc5be719249f474884b692445bafdd345ffd/packages/documentation-site/patternfly-docs/content/components/pagination/pagination.md?plain=1#L70